tests/mesh/test-merge-surface.cpp

```diff
@@ -66,25 +72,41 @@ void test()
         merged->nb_vertices() == 6, "[Test] Wrong number of vertices" );
     OPENGEODE_EXCEPTION(
         merged->nb_polygons() == 4, "[Test] Wrong number of polygons" );
+    for( const auto p : geode::Range{ merged->nb_polygons() } )
+    {
+        for( const auto e : geode::LRange{ 3 } )
+        {
+            merged->polygon_adjacent_edge( { p, e } );
+        }
+    }
```

Comment:
I'm not sure this double loop does anything ... is there an exception missing ?

Reply:
There are internal tests but i do not know if some compiler will remove this instruction.